docs/user-guide.md

```diff
@@ -24,34 +25,55 @@ This guide provides detailed instructions on how to configure your Liberty proje
-- Configure the [Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven#configuration) or [Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle#adding-the-plugin-to-the-build-script). We recommend using recent versions of the plugins as they include several important bug fixes.
+- [Configure the Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven#configuration) or [configure the Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle#adding-the-plugin-to-the-build-script). We recommend using the most recent versions of the plugins as they include several important bug fixes.
+
+    - [Liberty Maven Plugin latest release](https://github.com/OpenLiberty/ci.maven/releases/latest)
+
+    - [Liberty Gradle Plugin latest release](https://github.com/OpenLiberty/ci.gradle/releases/latest)
+
+### Configure your Java
 
-  The following versions are recommended at minimum:
-  - Liberty Maven Plugin -> 3.7.1
-  - Liberty Gradle Plugin -> 3.5.1
+Liberty Tools for Visual Studio Code requires Java 17 or later to ensure that the following language servers start properly:
+- [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server)
+- [Eclipse Language Server for Jakarta EE](https://github.com/eclipse/lsp4jakarta)
+- [XML Language Server (LemMinX)](https://github.com/eclipse/lemminx) 
 
-### Settings
+A toast message alerts you if any language server fails to run or if Visual Studio Code has trouble locating your Java.
 
-Liberty Tools for Visual Studio Code requires Java 17 or later to ensure that the [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server) and the [Eclipse Language Server for Jakarta EE](https://github.com/eclipse/lsp4jakarta) start properly. A toast message alerts you if either language server fails to run.
+<img src="/docs/screenshots/update%20jdk%20toast.png" width="40%"/> 
+<img src="/docs/screenshots/java_17_toast_alert.png" width="40%" />
 
-<img src="/docs/screenshots/java_17_toast_alert.png" width="50%" height="50%">
+To resolve this issue, you can define properties in your Visual Studio Code [settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson) file or set system environment variables to point Liberty Tools to your Java 17 or later. 
 
-To resolve this conflict, define `java.jdt.ls.java.home` and `xml.java.home` in your Visual Studio Code [settings.json](https://code.visualstudio.com/docs/getstarted/settings) file to point Liberty Tools to your Java 17 or later runtime.
+For both Liberty Config Language Server and Eclipse Language Server for Jakarta EE, Liberty Tools for Visual Studio Code will check for the Java versions in the following order:
+1. The [embedded JRE](https://github.com/redhat-developer/vscode-java#java-tooling-jdk) included by [Language Support for Java(TM) by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java). 
```

Comment:
Based on the latest info, is the embedded JRE still number 1?
Wondering how we should structure this section now...